src/app/dynamic/enterprise/quotas/project-quota-dialog/component.ts

```diff
@@ -149,16 +178,31 @@ export class ProjectQuotaDialogComponent implements OnInit, OnDestroy {
         validators: Validators.required,
         nonNullable: true,
       }),
-      subjectName: this._builder.control(
-        {value: subjectName, disabled: true},
-        {validators: Validators.required, nonNullable: true}
-      ),
+      subjectName: this._builder.control(subjectName ?? '', {
+        validators: Validators.required,
+        asyncValidators: AsyncValidators.ProjectSelection,
```

Comment:
I think we don't need this validator anymore since we've now switched to `km-combobox`